packages/bottender/src/bot/Bot.ts

```diff
@@ -9,12 +9,19 @@ import Context from '../context/Context';
 import MemoryCacheStore from '../cache/MemoryCacheStore';
 import Session from '../session/Session';
 import SessionStore from '../session/SessionStore';
-import { Action, Plugin, Props } from '../types';
+import {
+  Action,
+  BodyType,
```

Comment:
Why we need to add `-Type` suffix for those types?

Reply:
I'm drunk